### ISSUE TITLE
Unique Visitors for Acquisitions reports don't show any data for periods other than Day

### ISSUE THREAD

**Issue by Starker3** [piwik/piwik#20385]
This seems potentially related to #20216 

For the Acquisitions reports it seems that no matter what settings you have for the Unique visitor metrics, they are not shown for periods other than `Day`, even with the correct config settings applied:
```
enable_processing_unique_visitors_day = 1
enable_processing_unique_visitors_week = 1
enable_processing_unique_visitors_month = 1
enable_processing_unique_visitors_year = 1
enable_processing_unique_visitors_range = 1
```

There are no Unique Visitor Metrics displayed: 
![image](https://user-images.githubusercontent.com/20905821/220201397-21c22108-6931-4220-a0a6-8b7d00e10809.png)

They only display for the `Day` period: 
![image](https://user-images.githubusercontent.com/20905821/220201461-2b332060-76a1-490c-937c-4e1c28768140.png)

**Comment by bx80** [comment]
Thanks for the extra info @Starker3, I expect this will need to be fixed alongside #20216

**Comment by mattab** [comment]
Thanks for creating the issue. This is actually expected behavior / limitation that Unique Visitors only show for "Day" report. 

Actually we already have this FAQ: https://matomo.org/faq/how-to/faq_113/

It says:

> About the “Unique Visitors” column you can see in some reports: when you view a “Day” report, you can also find a “Unique Visitors” column in most reports. For example the number of unique visitors who reached your website from a specific Website Referrer, or the number of unique visitors who use a particular device or browser. This “Unique Visitors” column is only available in “Day” reports and unfortunately it cannot be added in the Week/Month/Year/Custom Date Range reports.

Maybe we could improve some in-app documentation on this? 

On the hand,  https://github.com/matomo-org/matomo/issues/20216 is a bug and will be fixed.

For now I'll close this one but feel free to create a new separate issue if you have any thoughts on how to better document this.
